src/components/MyTeachersComponents/NewEventStepper.tsx

```diff
@@ -169,26 +177,22 @@ export default function NewEventStepper(props: Props): React.ReactElement {
                   setDate(date);
                 }}
               />
+              <TimePicker
+                variant="inline"
+                label="Minutes and seconds"
```

Comment:
To keep consistency with previous input version let's change this label to "Time"
```suggestion
                label="Time"
```